tests/test_logic.py

```diff
@@ -394,3 +393,32 @@ def test_finite(op, data, dtype):
     dpnp_res = getattr(dpnp, op)(x, out=dp_out)
     assert dp_out is dpnp_res
     assert_equal(dpnp_res, np_res)
```

Comment:
What is about tests in SYCL queue and USM type scopes?

Reply:
I have added new tests for logic functions
Thank you!